etc/delphix-platform/ansible/apply

```diff
@@ -34,6 +35,7 @@ function apply_playbook() {
 	if ! ansible-playbook -vvv \
 		-c "$DLPX_ANSIBLE_CONNECTION" \
 		-i "$DLPX_ANSIBLE_INVENTORY" \
+		-e "root_filesystem=${ROOT_FILESYSTEM}" \
```

Comment:
I'm a late to the party, but we might want to start prefixing ansible variable names with `dlpx_` or `delphix_` since (AFAIK) there isn't any namespacing built into the variable names, so we could potentially conflict with other variables. E.g. for built in variables, iirc, they're prefixed with `ansible_`.. so in this case, something like `delphix_root_filesystem` is what I'm thinking.

This already landed, and is probably fine, but just wanted to make a note if we end up adding more variables in the future.